src/test/timedata_tests.cpp

```diff
@@ -3,6 +3,7 @@
 // file COPYING or http://www.opensource.org/licenses/mit-license.php.
 //
 #include <netaddress.h>
+#include <arpa/inet.h>
```

Comment:
You could include `compat.h` instead.

Reply:
Hi. Appveyor still fails with the same message. Any other suggestion?

Reply:
Hi. A quick update. 

compat.h includes the appropriate include files for windows. Unfortunately inet_pton has a slightly differen signature in win32. Any hints how to handle this without having a windows machine at hand?. Any suggestions are welcome, since this keeps me from completing the contract tests.

Thx.

Reply:
You can rebase your PR on #14922. That PR can fix the problem.